cloudgen_firewall/README.md

```diff
@@ -0,0 +1,81 @@
+# Agent Integration: cloudgen_firewall
+
+## Overview
+
+This integration monitors [cloudgen_firewall][1].
+
+## Setup
+
+### Installation
+
+The cloudgen_firewall check is included in the [Datadog Agent][2] package.
+Barracuda cloudGen Firewall installation is needed on your server.
+
+### Configuration
+
+1. Barracuda CloudGen Firewall with Administrative access.
+2. DataDog Agent installed and running(server or container that can receive syslog).
+3. Network Access between firewall and the dataDog agent (usually port 514 or custom).
+4. Syslog support enabled in Datadog Agent (tcp/udp listener configured).
+
+### Validation
+
+1. Confirm Datadog Agent is listening on the right port (eg., 514)
+    sudo netstat -tunlp | grep 514
+OR, for TCP/UDP listeners:
+    sudo lsof -i :514
+2. Confirm logs are reaching to the Agent and check for specific log source.
+    tail -f /var/log/datadog/syslog.log
+(If file doesn't exists, verify that syslog logs are being written by your config).
+3. Use tcpdump to confirm network traffic. On datadog Agent host:
+    sudo tcpdump -i any port 514
+You should see traffic from the cloudGen Firwewall IP address. if not, check firewall rules between cloudGen and Agent. Confirm the correct protocol (UDP/TCP) is being used on both sides.
+4. Check Live Tail in Datadog. Filter by source and service as defined in conf.yaml.
+5. Create a Test Log on the firewall by triggering an event.
+6. Check parsing and tagging in datadog.
+7. Use a Dashboard that is created to track volume over time.
+
+## Data Collected
+
+### Metrics
+
+cloudgen_firewall does not include any metrics.
+
+### Log Collection
+
+
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in the `datadog.yaml` file with:
+
+    ```yaml
+      logs_enabled: true
+    ```
+
+2. Add this configuration block to your `cloudgen_firewall.d/conf.yaml` file to start collecting your cloudgen_firewall logs:
+
+    ```yaml
+      logs:
+        - type: file
+          path:  /var/log/cloudgen_firewall.log
+          source: cloudgen_firewall
+          service: <SERVICE_NAME>
+    ```
+
+    Change the `path` and `service` parameter values and configure them for your environment.
+
+3. [Restart the Agent][4].
+
+### Events
+
+The cloudgen_firewall integration include recognizable events like messages (eg. login failed, rule hits). These can be surfaced as datadog events with parsing.
```

Comment:
```suggestion
The cloudgen_firewall integration includes log events such as failed logins and rule hits.
```

Reply:
Since this seems to be a description of the log events generated by the integration, this information would be a good fit for the `### Log collection` section. If the integration does not produce any Datadog events by default, please indicate that `cloudgen_firewall does not include any events`, or, alternatively, provide users with guidance or a link to transform their log events into Datadog events with parsing.